lib/desired-caps.js

```diff
@@ -346,6 +346,9 @@ const desiredCapConstraints = /** @type {const} */ ({
   enforceAppInstall: {
     isBoolean: true,
   },
+  triggerInputEventAfterSendkeys: {
```

Comment:
the new capability should be properly documented

Reply:
ok, documentation added.